guides/being-a-tech-lead-at-dxw.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Being a tech lead at dxw
+---
+
+## Being a tech lead...
+
+### ...at an agency
+
+Our work with a client is always temporary. Sometimes we know the end before we
+start, and sometimes it seems like it will go on forever. But eventually the
+relationship will end.
+
+As an agency, our clients bring us in when there's something they can't do or
+something they don't have the time to do.
+
+#### Working with clients short of expertise
+
+If our client can't do the thing, then you're explicitly there as the expert.
+You're there because you're the one who knows how to build the software they
+need. That means when you tell the client how to build it, they'll listen.
+
+They might be trying to build their own in-house capability; if so they'll be
+looking to you to help them learn. Yes, you're there to deliver working software
+that meets the needs of their users. But you're also there to help them get
+better. When we leave, we need to leave them with working software and an
+in-house team that has the expertise to maintain and develop it.
+
+However, they might not be trying to build their own team. Although software is
+never done, we need to try our best to get there by the time we leave. Their
+service needs to be:
+
+- maintainable with minimal effort
+- reliable, as there's no standing team to respond to issues
+- feature-complete enough that their users can do everything they need without
```

Comment:
I think "feature-complete" and "everything" could be interpreted as something that contradicts with "never done". This point sounds like it's trying to address user needs and usability, perhaps something like: "enabling users to deliver the minimum viable set of outcomes with a user experience that is good enough"?

Reply:
Good point. Perhaps better phrased as "feature-complete enough and with a good enough user experience to meet their users' core needs"?

Reply:
Using Dom's suggested wording here.